optaplanner-examples/src/test/java/org/optaplanner/examples/conferencescheduling/solver/ConferenceSchedulingScoreRulesXlsxTest.java

```diff
@@ -0,0 +1,228 @@
+
+            nextRow(false);
+            readHeaderCell("Constraint package");
+            currentPackage = nextStringCell().getStringCellValue();
+            nextRow(false);
+            readHeaderCell("Constraint name");
+            currentConstraint = nextStringCell().getStringCellValue();
+            nextRow(false);
+            readHeaderCell("Score");
+            expectedScore = HardSoftScore.parseScore(nextStringCell().getStringCellValue());
+
+            ConferenceSolution nextSheetSolution = solutionCloner.cloneSolution(initialSolution);
+            Map<String, Talk> talkMap = nextSheetSolution.getTalkList().stream().collect(
+                    Collectors.toMap(Talk::getCode, Function.identity()));
+
+            logger.info(currentSheet.getSheetName());
+            scoreVerifier.assertHardWeight(currentPackage, currentConstraint, unassignedScore.getHardScore(), nextSheetSolution);
+            scoreVerifier.assertSoftWeight(currentPackage, currentConstraint, unassignedScore.getSoftScore(), nextSheetSolution);
+
+            nextRow();
+            readTimeslotDays();
+            nextRow(false);
+            readHeaderCell("Room");
+            readTimeslotHours();
+            while (nextRow()) {
+                String roomName = nextStringCell().getStringCellValue();
+                Room room = roomMap.get(roomName);
+                if (room == null) {
+                    throw new IllegalStateException(currentPosition() + ": The room (" + roomName
+                            + ") does not exist in the room list.");
+                }
+                for (int i = 0; i < columnIndexToStartTimeMap.size(); i++) {
+                    String talkCode = nextCell().getStringCellValue();
+                    if (!talkCode.isEmpty()) {
+                        Talk talk = talkMap.get(talkCode);
+                        if (talk == null) {
+                            throw new IllegalStateException(currentPosition() + ": Talk (" + talkCode
+                                    + ") does not exist in the talk list.");
+                        }
+
+                        LocalDateTime startDateTime = LocalDateTime.of(columnIndexToDateMap.get(currentColumnNumber), columnIndexToStartTimeMap.get(currentColumnNumber));
+                        LocalDateTime endDateTime = LocalDateTime.of(columnIndexToDateMap.get(currentColumnNumber), columnIndexToEndTimeMap.get(currentColumnNumber));
+
+                        Timeslot timeslot = timeslotMap.get(Pair.of(startDateTime, endDateTime));
+                        if (timeslot == null) {
+                            throw new IllegalStateException(currentPosition() + ": The timeslot with date (" + startDateTime.toLocalDate().toString()
+                                    + "), startTime (" + startDateTime.toLocalTime().toString() + ") and endTime (" + endDateTime.toLocalTime().toString()
+                                    + ") that doesn't exist in the other sheet (Timeslots).");
```

Comment:
This statement could uses some new lines to make it better visually organized and fit into the 128 chars line length limit.

Also, the `toString()` calls are redundant.